src/plugins/data_views/common/data_views/data_views.ts

```diff
@@ -505,6 +511,37 @@ export class DataViewsService {
     return fields;
   };
 
+  /**
+   * Get existing index pattern list by providing string array index pattern list.
+   * @param indices index pattern list
+   * @returns index pattern list
+   */
+  getExistingIndices = async (indices: string[]): Promise<string[]> =>
+    await Promise.all(
+      indices.map(async (pattern) => {
+        try {
+          // when checking a negative pattern, check if the positive pattern exists
+          const indexToQuery = pattern.trim().startsWith('-')
+            ? pattern.trim().substring(1)
+            : pattern.trim();
+          const res = await this.getFieldsForWildcard({
+            // check one field to keep request fast/small
+            fields: ['_id'],
```

Comment:
I don't think there is a fields property for `getFieldsForWildcard`: https://github.com/stephmilovic/kibana/blob/d59d0c92cad2331dfda39e69f8807ccd964c2dd9/src/plugins/data_views/common/types.ts#L123